src/Bundle/Maker/MakeGrid.php

```diff
@@ -0,0 +1,285 @@
+            $fieldData['options'] = [];
+            $fieldData['options']['template'] = $io->ask('Enter the twig template path', null);
+        }
+
+        if ('datetime' === $fieldData['type']) {
+            $fieldData['options'] = [];
+            $fieldData['options']['format'] = $io->ask('Enter the date format', 'Y-m-d H:i:s');
+        }
+
+        return [
+            $fieldName => $fieldData,
+        ];
+    }
+
+    private function askForNextAction(ConsoleStyle $io, string $section)
+    {
+        $io->writeln('');
+
+        $actionTypes = static::$defaultActionTypes[$section];
+
+        if (count($actionTypes) > 1) {
+            $choiceQuestion = new ChoiceQuestion(
+                'Enter the action types (coma-separated)',
+                static::$defaultActionTypes[$section],
+                implode(', ', static::$defaultActionTypes[$section])
+            );
+            $choiceQuestion->setMultiselect(true);
+            $choiceQuestion->setAutocompleterValues($actionTypes);
+            $actionTypes = $io->askQuestion($choiceQuestion);
+        }
+
+        $data = [];
+        foreach ($actionTypes as $type) {
+            $data[$type] = [
+                'type' => $type,
+            ];
+        }
+
+        return $data;
+    }
+
+    private function generateGridConfigFile(
+        InputInterface $input,
+        ConsoleStyle $io,
+        Generator $generator,
+        array $fields,
+        array $actions
+    ): void {
+        $section = $input->getArgument('section');
+        $resourceAlias = $input->getArgument('resource');
+        [$appName, $resourceName] = $this->resourceHelper->splitResourceAlias($resourceAlias);
+        $gridId = sprintf('%s_%s_%s', $appName, $section, $resourceName);
+
+        $gridConfigDir = $this->getGridConfigDir($resourceAlias, $section);
+        $modelClass = $this->resourceHelper->getResourceModelFromAlias($resourceAlias);
+
+        $gridData = [
+            'driver' => [
+                'name' => 'doctrine/orm',
+                'options' => [
+                    'class' => '"%'.$modelClass.'%"',
+                ],
+            ],
+        ];
+
+        if (count($fields) > 0) {
+            $gridData['fields'] = $fields;
+        }
+
+        if (count($actions) > 0) {
+            $gridData['actions'] = $actions;
+        }
+
+        $data = [
+            'sylius_grid' => [
+                'grids' => [
+                    $gridId => $gridData,
+                ],
+            ],
+        ];
+
+        $yaml = Yaml::dump($data, 10, 4, Yaml::DUMP_NULL_AS_TILDE);
+
+        $this->fileSystem->dumpFile($gridConfigDir, $yaml);
+    }
+
+    private function getGridConfigDir(string $resourceAlias, string $section)
+    {
+        $resource = $this->resourceHelper->getResourceNameFromAlias($resourceAlias);
+        $filename = $resource.'.yaml';
+
+        return sprintf('%s/%s/%s', $this->projectDir.'/config/packages/grids', $section, $filename);
```

Comment:
I'm not sure about adding `grid` subfolder to packages, as I found it confusing with `env` folders

Reply:
I think this path should be configurable on the bundle.
We can also ask for it on command when it's not configured on the bundle configuration.

Reply:
But you're right, by default, I should move them to config/grids directory (On Monofony project too).